packages/scheduler/src/forks/SchedulerFeatureFlags.www.js

```diff
@@ -10,13 +10,12 @@
 // $FlowFixMe[cannot-resolve-module]
 const dynamicFeatureFlags = require('SchedulerFeatureFlags');
 
-export const {
-  userBlockingPriorityTimeout,
-  normalPriorityTimeout,
-  lowPriorityTimeout,
-  enableRequestPaint,
-} = dynamicFeatureFlags;
+export const {enableRequestPaint} = dynamicFeatureFlags;
 
-export const frameYieldMs = 10;
-export const enableSchedulerDebugging = true;
+export const enableSchedulerDebugging = false;
 export const enableProfiling = __DEV__;
+export const frameYieldMs = 10;
```

Comment:
@noahlemen remind me, did we decide if 10ms was going to be the default everywhere?

Reply:
looking back at prior discussions, yes, it doesn't seem like there was any suggestion to change it, just possible followups regarding `requestPaint`